MemberExpectations.md

```diff
@@ -9,16 +9,14 @@ contributors are expected to be respectful of these different viewpoints,
 and to work collaboratively with one another in a manner that constructively
 elevates all contributors.
 
-When an individual contributor chooses to accept an invitation to participate
-in a leadership role, they implicitly take on an additional obligation to protect
-and be respectful of both the project and all other contributors.  When
```

Comment:
I think the key part of this was "being respectful of other contributors".  Respecting a decision that has been made by the contributors as a group is part of that.  I'm not sure that the link between project stewardship and respecting group decisions is as clear to me as the previous wording.   We could just remove 'protect' instead.

Reply:
@mhdawson I get what you're saying. Taken in total, IMO the link between project stewardship and being respectful of group decisions is pretty clear in the full text:

> When someone takes on a leadership role, they take on additional obligations of
project stewardship. When decisions are made within the established guidelines
and policies of the project, those in leadership roles have a responsibility to
uphold and respect the decision even if they disagree with it. This is
especially important in external communications, for example in social media.
Should the member be unwilling or unable to do so, then they should resign their
leadership position. This does not mean that decisions cannot be revisited and
discussed within the team at a later time.

"Being respectful of other contributors" already shows up all over this document (as it should):

* First paragraph: Code of Conduct compliance in the first paragraph
* Second paragraph: "All contributors are expected to be respectful of these different viewpoints, and to work collaboratively with one another in a manner that constructively
elevates all contributors."
* Fourth paragraph: "Members of our leadership groups must also conduct themselves in a
professional and respectful manner."
* Nearly every (if not every!) bullet point in the 9-item list that follows the fourth paragraph.